.github/workflows/bot-proxy.yml

```diff
@@ -0,0 +1,47 @@
+name: GitHub Bot Proxy
+
+on:
+  # Watch for any completed run on bot.yml workflow
+  workflow_run:
+    workflows: [GitHub Bot]
+    types: [completed]
+
+jobs:
+  # This workflow monitors any run completed on the GitHub Bot workflow and
+  # checks if the event that triggered it is limited to read-only permissions
+  # (e.g 'pull_request_review' on a pull request opened from a fork).
+  # In this case, it reruns the GitHub Bot workflow using a 'workflow_dispatch'
```

Comment:
```suggestion
  # In this case, it reruns the GitHub Bot workflow using a 'workflow_run'
```

Reply:
No it was correct, it reruns it using a `workflow_dispatch`